SimPEG/EM/Analytics/EMcircuit.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import division
+import numpy as np
+from scipy.constants import mu_0, pi, epsilon_0
+from scipy.special import erf
+from SimPEG import Utils
+
+def Qfun(R, L, f, alpha=None):
+    if alpha is None:
+        omega = np.pi*2*f
+        tau = L/R
+        alpha = omega*tau
+    Q = (alpha**2+1j*alpha) / (1+alpha**2)
+    return alpha, Q
+
+def Mijfun(x,y,z,incl,decl,x1,y1,z1,incl1,decl1, area=1.,area0=1.):
+    """
+        Compute mutual inductance between two loops
+
+        This
+
+        Parameters
```

Comment:
for the docs, can you follow the sphinx conventions ? The mesh documentation is a good example. (This way the docs will actually be linked, for example, to numpy where you can see what type of input is expected) 
```
:param numpy.ndarray name: description
:rtype: numpy.ndarray
:return: what is returned 
```

![image](https://cloud.githubusercontent.com/assets/6361812/17459375/80d621c8-5be9-11e6-9b4c-9450ad7d70f5.png)